doc/langref.html.in

```diff
@@ -11040,20 +11040,26 @@ fn readU32Be() u32 {}
       {#header_close#}
 
       {#header_open|Grammar#}
-      <pre><code>Root &lt;- skip ContainerMembers eof
+      <p>
+      This content is sourced from the <a href="https://github.com/ziglang/zig-spec">zig-spec</a> project.
+      It is not necessarily up to date. Please make contributions if you find a problem.
+      </p>
+      <!-- $$BEGIN GENERATED GRAMMAR$$ --><pre><code>Root &lt;- skip ContainerMembers eof
```

Comment:
we already have a meta language inside here, can you use that instead of HTML comments?

Reply:
good question. that would mean embedding the contents of a file from another repo into this html document with some html escaping. is there a precedent for doing that? can you point me at the tool that generates the docs?